### ISSUE TITLE
Beautifying the UI

### ISSUE THREAD

**Issue by Asutosh989** [Asutosh989/twitter-splitter#4]

**Comment by RitikPatni** [comment]
Can you provide more info

**Comment by Asutosh989** [comment]
@RitikPatni You can check the frontend and suggest and make changes as per your desire.

**Comment by RitikPatni** [comment]
Can you provide me a quick start guide??

**Comment by Harkindey** [comment]
Started this tho @RitikPatni

**Comment by RitikPatni** [comment]
Figured out how to start but can't really understand the functionality. @Harkindey can you provide me some insights

**Comment by Asutosh989** [comment]
@Harkindey Do you want to work on this?

**Comment by Harkindey** [comment]
@RitikPatni no bro

**Comment by thedammyking** [comment]
@RitikPatni I tried running this after cloning the repo and got a "TypeError: OAuthStrategy requires a consumerKey option"

**Comment by Priyansh2** [comment]
@Asutosh989 is this issue still open ?

**Comment by Asutosh989** [comment]
@Priyansh2 yes the issue is still open. If you want you can work on it. And feel free to ask if you need any details.

**Comment by Shubham-SK** [comment]
Hi, I could work on this, are you looking for something in particular?

**Comment by Asutosh989** [comment]
@altruisticnarcissist you can work on this, I have no specification for now as I am trying to resolve the backend issues. Fork and clone the repo and you can suggest any changes in the frontend.

**Comment by email2vimalraj** [comment]
If no one looks into it, can I start working on this?

**Comment by Asutosh989** [comment]
@email2vimalraj Yes you can start working on this.

**Comment by kushaggarwal** [comment]
Is this issue still open
